[Chore] Update README.md (#109)
nimblehq/github-notifications-mac: README.md

# Github Notifications Application

## What is Github Notifications Application?

> Github Notifications is a macOS application to manage Github Notifications to be more personalized. Receive notifications on your Mac when there are new updates on subscribed repositories.

Reviewer: ```suggestion
Github Notification Application is a macOS application for managing and personalized Github's notifications.
```

Reviewer: In the second sentence, I suggest you list down the actions that trigger to show notification, f.e: mention, request review, etc.

> Github Notification Application is a macOS application for managing and personalized Github's notifications. Receive notifications from subscribed repositories on your Mac when you were assigned, mentioned, review requested, and when your pull request was reviewed.

Reviewer 2: ```
Github Notification Application is a macOS application for managing and personalized Github's notifications. 
It receives notifications from subscribed repositories on your Mac when you were assigned, mentioned, review requested, and when your pull request was reviewed.
```
What do you think? 🤔
